### DISCUSSION TITLE
What are the bareminimum configuration needed for LDAP to work?

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8088] [Q&A]
I have set the following, but not seeing any change:
INVENTREE_LDAP_ENABLED (enabled)
INVENTREE_LDAP_SERVER_URI (added url of my own)
Any other variable that needs to be setup? And what should I expect if this is set up properly?

**Comment by matmair** [comment]
This depends entirely on your LDAP server. Most will at least need bind credentials.
The UI does not change, valid credentials from your LDAP server will just start working and provision new accounts once users sign in.
https://docs.inventree.org/en/stable/start/advanced/#ldap
